styles/styles.css

```diff
@@ -13,10 +13,11 @@ svg > line {
 
 /* FLEX LAYOUT  */
 .mainWrapper {
-  height: 100%;
+  min-height: 100%;
   display: flex;
   justify-content: space-between;
   flex-direction: column;
+  position: relative;
```

Comment:
We can define all of these properties by bootstrap classes. What do you think about it?

Reply:
Currently we are switching to React and as far as I'm concerned there are some bootstrap issues (invisible dropdowns etc.). That's why I suggest leaving it in pure css to avoid compatibility issues so far.

Reply:
@oskarhinc @afijal What do you think about it?

Reply:
We will probably have to switch all those dropdowns to bootstrap, and the issues were rather with jQuery, so @dserkowski is right.

Reply:
ok I will try to extract bootstrap classes

Reply:
Just to clarify, problems are connected to popper which is used by some bootstrap components like dropdown @pawelcieszko @oskarhinc